src/hub.js

```diff
@@ -195,10 +195,16 @@ const onReady = async () => {
     if (NAF.connection.adapter && NAF.connection.adapter.localMediaStream) {
       NAF.connection.adapter.localMediaStream.getTracks().forEach(t => t.stop());
     }
-    hubChannel.disconnect();
+    if (hubChannel) {
+      hubChannel.disconnect();
+    }
     const scene = document.querySelector("a-scene");
-    scene.renderer.animate(null); // Stop animation loop, TODO A-Frame should do this
-    document.body.removeChild(scene);
+    if (scene) {
+      if (scene.renderer) {
+        scene.renderer.animate(null); // Stop animation loop, TODO A-Frame should do this
+      }
```

Comment:
is there cleanup to be done for `registerInputActions`?

Reply:
In theory we could define an empty input mapping, but I'm not sure it would make much of a difference in terms resource usage. The input mapping system will stop ticking when the scene stops ticking anyway, which stops with the renderer loop.